modules/cli/src/main/java/org/apache/ignite/internal/cli/core/repl/completer/NodeUrlProvider.java

```diff
@@ -40,7 +40,7 @@ public NodeUrlProvider(Session session, ConfigManagerProvider configManagerProvi
     }
 
     /** Resolves the url for given words. */
-    public String resolveUrl(String[] words) {
+    public String resolveUrl(String... words) {
```

Comment:
What the point of vararg here? So, lets imagine that its required change but why you didn't change usages of this method. For example org/apache/ignite/internal/cli/core/repl/completer/hocon/ClusterConfigDynamicCompleterFactory.java:58 still use String array initialization